schema.graphql

```diff
@@ -630,6 +630,7 @@ enum CreateDatasetVersionError {
 
 enum CreateDatasetVersionFileError {
   VERSION_NOT_FOUND
+  LOCKED_VERSION
```

Comment:
Not sure this should be in this PR

Reply:
Hum, that's weird because it should be on the main branch. Let me check